### ISSUE TITLE
Basis set is different from EMSL library basis

### ISSUE THREAD

**Issue by HansolP** [pyscf/pyscf#484]
I found that some basis sets are different from one in EMSL basis library
One example is cc-pvdz basis.

pyscf/gto/basis/cc-pvdz.dat

![image](https://user-images.githubusercontent.com/58722204/71776943-d1f49880-2fdc-11ea-9146-4dd01816bb53.png)

But, EMSL library's cc-pvdz is
![image](https://user-images.githubusercontent.com/58722204/71776955-edf83a00-2fdc-11ea-9517-9c5c7b8abcd3.png)

Contracted basis is different.
I glance over some other basis set, cc-pvtz is also different from EMSL library.
Since it is enough difference to affect simulation results, I think it is necessary to update basis set in gto/basis.

**Comment by gkc1000** [comment]
Hi, the basis sets in PySCF *are* obtained from EMSL. Note that the two
basis sets that you have shown exactly span the same space, but one of the
1s functions (which appears in 2s) is removed from 1s in the PySCF basis.
They will produce exactly the same energies. (Did you check your statement
that simulation results are affected?)

The shuffling of Gaussian functions is the result of "optimizing" the
contractions to speed up integral evaluation. On the EMSL basis set
exchange, you can choose the option "optimize general contractions" when
downloading the basis (it used to be an option on the main page, now you
have to click advanced), then you will obtain the same basis as PySCF.

**Comment by HansolP** [comment]
I’m sorry uploading the issue without deep understand of EMSL library, and I checked there is no ground state energy difference between optimized and not-optimized basis sets.
When I evaluated some physical values in real space, I found nice difference between two basis sets.
If I should use not-optimized basis sets, I would change basis sets personally.

I apologize for my mistakes and thank you for your clear explanation.
